frontend/packages/kubevirt-plugin/src/plugin.ts

```diff
@@ -77,6 +103,26 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).then((m) => m.VirtualMachinesDetailsPage),
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      path: `/k8s/ns/:ns/vmtemplates`,
+      loader: () =>
+        import(
+          './components/vm-templates/vm-template' /* webpackChunkName: "kubevirt-vmtemplates" */
+        ).then((m) => m.VirtualMachineTemplatesPage),
+    },
+  },
+  {
+    type: 'Page/Route',
+    properties: {
+      path: `/k8s/all-namespaces/vmtemplates`,
```

Comment:
If you need multiple paths associated with the same page component, you can do this:

```ts
{
  type: 'Page/Route',
  properties: {
    path: ['/k8s/all-namespaces/vmtemplates', '/k8s/ns/:ns/vmtemplates'],
    loader: () => {/* dynamically load VirtualMachineTemplatesPage */},
  },
},
```

There's no need to use string literals (back-ticks) if the value doesn't contain any expressions, using `'` for static strings is enough.

Also, consider adding `exact: true` to denote that we want an exact URL match.

Please check out the DevConsole plugin (`packages/dev-console/src/plugin.tsx`) and how it uses `Page/Route` extensions.

Reply:
Thanks !